### ISSUE TITLE
provide page-fade directive as control

### ISSUE THREAD

**Issue by x1B** [LaxarJS/laxar_uikit#113]
Having removed the directive `axPageFade` from LaxarJS core with LaxarJS/laxar#181 for LaxarJS `v1.0.0`. We should provide a replacement in LaxarJS UiKit that people can opt-in to.

**Comment by x1B** [comment]
### Usage Information

The `axPageFade` directive which was previously available with LaxarJS core (see LaxarJS/laxar#180) has been added as a control to LaxarJS UiKit. It displays a layer until the `didNavigate` event is received.

Typically, this control is not loaded from a widget, but directly included into the application markup, such as the `index.html` and `debug.html` files in the application template. The `style` is added inline to be in effect before scripts, layouts, widgets and theme styles are processed:

```html
<div data-ax-page-fade
    style="background: #fff; z-index: 100; position: fixed; left: 0; top: 0; right: 0; bottom: 0;">
</div>
```

For the directive definition to be available, your `init.js` needs to be augmented by something like this:

```js
/** … */
require( [
   …,
   'laxar_uikit/controls/page_fade'
], function( ..., axPageFadeModule ) {
   'use strict';

   widgetModules.angular = widgetModules.angular || [];
   widgetModules.angular.push( axPageFadeModule );
   ax.bootstrap( widgetModules );

} );
```

**Comment by x1B** [comment]
Implemented on `master` (`v1.0.0`).
